src/Tribe/Attendees_Table.php

```diff
@@ -513,25 +513,51 @@ public function extra_tablenav( $which ) {
 		 */
 		add_thickbox();
 
+		$parent = 'admin.php';
+
+		/**
+		 * Filtert to show email form for non-admins.
+		 *
+		 * @since TBD
+		 *
+		 * @param boolean
+		 */
+		$allow_fe = apply_filters( 'tribe_allow_admin_on_frontend', false );
+		if ( $allow_fe && ! is_admin() ) {
+			global $wp;
+			$parent = $wp->request . '/';
```

Comment:
Should there be a untrailingslash() no the request property? Just asking since I don't use that property much.

Reply:
I don't believe it needs it but it wouldn't hurt.